src/main/resources/application.yaml

```diff
@@ -4,6 +4,23 @@ server:
 spring:
   application:
     name: weather-reporter
+  datasource:
+    url: jdbc:postgresql://localhost:5432/weather
+    driver-class-name: org.postgresql.Driver
+    username: admin
+    password: admin
+  jpa:
+    properties:
+      hibernate:
+        format_sql: true
```

Comment:
При поднятии контекста падает ошибка с таким диалектом, нашла информацию, что она deprecated с версии hibrnate 6.0.0, у меня подключена более поздняя версия.

Указываю явно диалект org.hibernate.dialect.PostgreSQLDialect, все поднимается без ошибки, но есть предупреждение, что он и так по дефолту используется:

: HHH90000025: PostgreSQLDialect does not need to be specified explicitly using 'hibernate.dialect' (remove the property setting and it will be selected by default)

Тогда указывать явно org.hibernate.dialect.PostgreSQLDialect или опустить эту проперти?